hawkular-alerts-metrics/src/main/java/org/hawkular/alerts/external/metrics/Expression.java

```diff
@@ -58,44 +60,22 @@
         min,
         /** Maximum reported value for the period. */
         max,
-        // Group metric only
-        /** number of distinct [trait] values for the group. No period. */
-        card(false),
-        /** percent [0.0,1.0] with last-reported availability down. No period */
-        down(false),
-        /** percent [0.0,1.0] with last-reported availability up. No period */
-        up(false);
-
-        private final boolean single; // true if supported by single metric/ requires a period
+        /** Count of UP avail for the period. */
+        heartbeat;
```

Comment:
I don't follow these changes.
Perhaps this is related to the changes on external alerter logic commented in some meeting.